appveyor.yml

```diff
@@ -20,22 +20,8 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
-      CONDA_PY: 27
 
     - TARGET_ARCH: x64
-      CONDA_PY: 27
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```

Comment:
@MSeifert04 unfortunately you are hitting a bug that is eluding us. Your re-rendering removed the build matrix and should not be merged. I will investigate this to try to find what is going on, but we saw this before...